### PR TITLE
release version 0.11.0

### LINKED ISSUE #298: release tracking issue
A lot of stuff has already changed and fixed since the last release. But there is still some stuff that has to be done.

### REQUIRED
- [x] decide on refactoring of error types (#295)
- [x] fix `codespan` readme (still has gitter badge)
- [x] increment version number (there are some breaking changes)
- [x] check changelog (most of the stuff should've been covered in c9f640bf7161258bc30b6f432bdbe2612c9bc1f0 already)
### OPTIONAL
- ~resolve #272~ (process could be tested with this release)
- [x] merge #297 so there is not another breaking version update right again

### PR DESCRIPTION
**Pull request by Johann150**
Incremented versions of all crates to `0.10.2` to sync them back up.
After a cursory check the changelogs seem to be okay. Only had to fix links for 0.9.5 in the codespan and codespan-reporting changelogs. Had already fixed that in the codespan-lsp changelog before.

- [ ] add the concrete release date

Closes #298

### CODE REVIEW

File: codespan-reporting/CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.10.2] - 2020-10-XX
+
 There is now a [code of conduct](https://github.com/brendanzab/codespan/blob/master/CODE_OF_CONDUCT.md)
 and a [contributing guide](https://github.com/brendanzab/codespan/blob/master/CONTRIBUTING.md).
```
Comment (brendanzab) [inline codespan-reporting/CHANGELOG.md:13]: Might be good to note that we have skipped `v0.10.0` and `v0.10.1` to get us back in sync with the `codespan-lsp` version

File: codespan/CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.10.2] - 2020-10-XX
+
```
Comment (brendanzab) [inline codespan/CHANGELOG.md:11]: Might be good to note the version jump here, like with `codespan-reporting`

Comment (Johann150) [reply codespan/CHANGELOG.md:11]: done for both

**Comment by brendanzab** [comment]
Probably need to now move to v0.11.0, given the changes to the public API in `codespan-lsp`.

**Pull request closed without merging**